papers/oleksandr_pavlyk/sklearn/sklearn.rst

```diff
@@ -28,33 +27,33 @@ We prepared several benchmarks to demonstrate performance that can be achieved w
 .. code-block:: python
 
         from __future__ import print_function
-        import numpy as np
         import timeit
         from numpy.random import rand
         from sklearn.cluster import KMeans
 
         import argparse
-        argParser = argparse.ArgumentParser(prog="pairwise_distances.py",
-                                            description="sklearn pairwise_distances benchmark",
-                                            formatter_class=argparse.ArgumentDefaultsHelpFormatter)
-
-        argParser.add_argument('-i', '--iteration', default='10', help="iteration", type=int)
-        argParser.add_argument('-p', '--proc', default=-1, help="n_jobs for algorithm", type=int)
+        argParser = argparse.ArgumentParser(
+            prog="kmeans_bench.py",
+            description="K-means benchmark")
+
+        argParser.add_argument('-i', '--iteration',
+              help="iteration", type=int, default=10)
+        argParser.add_argument('-p', '--proc', default=-1,
+              help="n_jobs for algorithm", type=int)
         args = argParser.parse_args()
```

Comment:
I'd actually say that in the paper the full source is an overkill, it should rather refer to github repo and show only important parts which are necessary for understanding. Approving now since it's of course better than what we have now